imap/http_caldav_sched.c

```diff
@@ -594,14 +594,20 @@ static int imip_send(const char *userid, struct sched_data *sched_data,
                      const char *sender, const char *recipient)
 {
     const char *notifier = config_getstring(IMAPOPT_IMIPNOTIFIER);
+    int r = 0;
+
+    struct buf myrecipient = BUF_INITIALIZER;
+    charset_decode_percent(&myrecipient, recipient);
```

Comment:
I prefer not to reuse the same variable name for data that has different levels of decoding, so if we called it `const char *rawrecipient` in the signature and then have a `const char *recipient` for the rest.

Reply:
thanks, fixed